pulseapi/profiles/serializers.py

```diff
@@ -84,6 +85,7 @@ class UserProfilePublicSerializer(UserProfileSerializer):
     """
     name = serializers.CharField(read_only=True)
     published_entries = serializers.SerializerMethodField()
+    created_entries = serializers.SerializerMethodField()
```

Comment:
Can we keep the property and its `def` paired in the code? (maybe move this to above L101 instead, so we have published+def, profile, created+def)